src/endpoints/endpoints.controller.ts

```diff
@@ -1,4 +1,18 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { FindApiEndpoints } from './dto/find-api-endpoints.dto';
+import { EndpointsService } from './endpoints.service';
 
+@ApiTags('endpoints')
 @Controller('endpoints')
-export class EndpointsController {}
+export class EndpointsController {
+    constructor(
+        private readonly endpointsService : EndpointsService
+    ){}
+
+    @Get('api:apiId')
```

Comment:
I do not understand the construction of the route '('api:apiId')'

Reply:
it goes /categories/api:apiId

There might be an endpoint to get a category by id. 

The route above is there to differentiate and make it less ambiguous.

Reply:
is it not supposed to go to **/categories/api/:apiID** the routing process is somewhat confusing

Reply:
the endpoint gets all the apis in an endpoint. 

The route is /endpoints/api/:apiId